src/amqp-connector.js

```diff
@@ -0,0 +1,66 @@
+const amqp = require('amqplib');
+class AmqpConnector {
+  constructor(app) {
+    this.app = app;
+  }
+
+  async sendToQueue(host, queue, message) {
```

Comment:
IMHO `host` should be passed as constructor argument, so that we don't need to re-connect every time we send something to the queue. A new method `connect` could be used to open the connection and save it in an instance variable. This method would be called just once when starting the app.

The same could be argued for the result of `createChannel()`. However, I'm no ampqlib expert, maybe channels may not be cached?